hieradata/role/openvpn.yaml

```diff
@@ -0,0 +1,37 @@
+---
+psick::profiles::linux_classes:
+  openvpn: openvpn
+  fw: psick::firewall::iptables
+
+
+psick::firewall::iptables::extra_rules:
```

Comment:
If you don't want to manage ipv6 add somewhere here:
psick::firewall::iptables::manage_ipv6: false

Remember to update the psick module to latest version.

Reply:
done with r10k command on the puppet server